wiki/Reporting_bad_behaviour/Abuse/id.md

```diff
@@ -1,76 +1,76 @@
+Setelah itu, blokir atau bisukan (*mute*) pengguna yang bersangkutan dengan mengeklik tombol titik tiga yang terdapat pada pojok kanan atas tweet yang sama melalui *browser* atau aplikasi Twitter milikmu. Lihat [panduan Twitter berikut](https://help.twitter.com/id/using-twitter/blocking-and-unblocking-accounts) untuk informasi lebih lanjut.
 
-Perlu diketahui bahwa orang tersebut akan melihat Anda memblokirnya jika mereka memeriksa profil Twitter Anda. Jika Anda khawatir hal ini akan memperburuk masalah, bisukan orang tersebut. Ini akan menghapus tweet mereka dari timeline Anda dan mencegah Anda melihat konten mereka lagi, namun mereka masih dapat melihat dan mengomentari semua tweet yang Anda buat.
+Harap diingat bahwa para pengguna yang telah diblokir tidak akan lagi dapat melihat isi profil Twittermu ke depannya. Apabila kamu khawatir hal ini hanya akan memperkeruh suasana, bisukan (*mute*) orang tersebut. Tindakan ini akan menyembunyikan tweet-tweet mereka dari *timeline*-mu tanpa menghalangi mereka untuk dapat melihat dan mengomentari tweet-tweet yang kamu kirimkan seperti pada umumnya.
 
-Jika tweet tersebut berasal dari akun yang ditautkan ke profil pemain osu! melalui situs web kami (muncul di bawah avatar mereka dan tanggal bergabung, di samping ikon burung), silakan kirim email ke [abuse@ppy.sh](mailto:abuse@ppy.sh) yang berisi salinan tweet dalam bentuk gambar jika memungkinkan, dan informasi lain yang dapat Anda pikirkan.
+Apabila tweet tersebut berasal dari akun Twitter yang tertaut pada laman profil osu! mereka masing-masing (yang terdapat di bawah avatar dan tanggal bergabung, di samping ikon burung), harap kirimkan email ke alamat [abuse@ppy.sh](mailto:abuse@ppy.sh) dengan menyertakan salinan atau *screenshot* tweet yang bersangkutan beserta informasi-informasi lain yang sekiranya dapat berguna.
 
-## Seorang anggota dari salah satu tim sukarelawan (NAT, GMT) mendekati/berbicara/bertindak secara tidak senonoh terhadap saya!
+## Salah seorang anggota tim sukarelawan (NAT, GMT) mendekati/berbicara/bertindak secara tidak senonoh terhadap saya!
 
-Kirim email ke [abuse@ppy.sh](mailto:abuse@ppy.sh) berisi informasi sebanyak mungkin **segera**.
+Harap kirimkan email ke alamat [abuse@ppy.sh](mailto:abuse@ppy.sh) **sesegera mungkin** dengan menyertakan informasi yang selengkap-lengkapnya.
 
-Jika sesuatu yang ingin Anda masukkan membuat Anda merasa malu, sensor saja. Ini termasuk tangkapan layar, log obrolan, apa pun yang dapat Anda lampirkan ke email.
+Apabila kamu ingin melampirkan suatu bukti (baik itu berupa *screenshot*, *chat log*, ataupun yang lainnya) namun kamu merasa malu atau tidak enak hati kepada pihak-pihak tertentu, kamu dapat menyensornya terlebih dahulu.
 
-Kami akan memeriksanya secepat mungkin.
+Kami akan memproses laporanmu dengan segera.
```

Comment:
```suggestion
Kami akan memroses laporanmu dengan segera.
```

Reply:
o actually i'm not sure which one is correct so i changed it into `menindaklanjuti` instead 👀